.github/workflows/python_tests.yml

```diff
@@ -111,7 +111,7 @@ jobs:
       - name: Install python
         uses: actions/setup-python@v4
         with:
-          python-version: ${{ matrix.params.py_ver }}
+          python-version: '${{ matrix.params.py_ver }}'
```

Comment:
I don't quite understand why we need quotes here. The input is already in quotes(a string) and this variable is able to catch it and run the python 3.10 tests.

Reply:
Ah, I was incorrect. I expected template rendering would make the `${{ matrix.params.py_ver }}` render to `3.10` without quotes, and that the following YAML parsing would lead to a failure.

It seems that my guess on how github expression's like `${{ }}` were replaced was incorrect, and a quote there wasn't needed. I knew that `echo '${{ steps.setup-python.outputs.python-version }}'` would translate to `echo '3.10.7'` - without quotes.